### PR TITLE
Use the proper Rizin version for a given rz-binding release

### PR DESCRIPTION
**Pull request by ret2libc**

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+    - name: Checkout rizin
+      uses: actions/checkout@v3
+      with:
+        repository: rizinorg/rizin
+        path: rizin
+        ref: ${{ github.ref }}
+
+    - name: Run rz-bindgen
+      uses: pypa/cibuildwheel@v2.8.0
```
Comment (XVilka) [inline .github/workflows/release.yml:30]: Maybe use just `@v2`? To avoid updating too often

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    runs-on: ubuntu-latest
+    needs: [run-bindgen]
+    steps:
+    - name: Download artifacts
+      uses: actions/download-artifact@v3
+      with:
+        name: artifact
+        path: dist
+    - name: Publish wheels
+      uses: pypa/gh-action-pypi-publish@v1.5.1
```
Comment (XVilka) [inline .github/workflows/release.yml:48]: Same

**Review by XVilka — APPROVED** [review]

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+jobs:
+  run-bindgen:
+    name: Run bindgen
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-20.04, windows-2019, macos-latest]
```
Comment (XVilka) [inline .github/workflows/release.yml:16]: Maybe use current LTS? 22.04 instead?

File: .github/workflows/release.yml (hunk 2 above)
Comment (ret2libc) [reply .github/workflows/release.yml:48]: I'm not sure about these. I see also the doc says to use the full version. I'd keep them as they are for now.

**Pull request merged**